Area cleanArea
---
int16 status